src/template/index.js

```diff
@@ -0,0 +1,248 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+/* eslint-disable no-param-reassign, no-underscore-dangle, import/no-extraneous-dependencies */
+const { Request } = require('node-fetch');
+// eslint-disable-next-line  import/no-unresolved
+const { main } = require('./main.js');
+/*
+ * Universal Wrapper for serverless functions
+ */
+
+/**
+ * Checks if the content type is binary.
+ * @param {string} type - content type
+ * @returns {boolean} {@code true} if content type is binary.
+ */
+function isBinary(type) {
```

Comment:
this list it more comprehensive:
https://github.com/adobe/openwhisk-action-utils/blob/1c4000bb178a6c56159e53c965bd04837327e274/src/expressify.js#L107-L175